src/pages/OpportunityDetail.tsx

```
@@ -0,0 +1,497 @@
+    mutationFn: () => {
+      if (!user) {
+        throw new Error('You must be logged in to express interest');
+      }
+      return expressInterest(user.id, id!, message);
+    },
+    onSuccess: () => {
+      toast("Interest submitted! The organization will contact you soon.");
+      setShowInterestDialog(false);
+      setMessage('');
+    },
+    onError: (error: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)